src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -377,7 +377,7 @@ private static VersionInfo GetVersionInfo()
 
         private readonly float[] _biases;
         private readonly VBuffer<float>[] _weights;
-        private readonly LinearModelStatistics _stats;
+        public readonly ModelStatisticsBase Statistics;
 
```

Comment:
Can this really be `Base`? What if use want to access statistics of the underlying linear function? I don't feel casting is a good idea.

Reply:
we are not casting anywhere in this file. Are you referring to the other one?

---
In reply to: [266660951](https://github.com/dotnet/machinelearning/pull/2048#discussion_r266660951) [](ancestors = 266660951)

Reply:
They happen outside this file. We do `ModelStatisticsBase`  as `LinearModelStatisticsBase` somewhere.